decidim-core/spec/mailers/reported_mailer_spec.rb

```diff
@@ -150,8 +151,39 @@ module Decidim
       end
 
       it "includes the reported content" do
-        expect(email_body(mail)).to match(reportable.title["en"])
-        expect(email_body(mail)).to match(reportable.body["en"])
+        expect(email_body(mail)).to match(translated(reportable.title))
+        expect(email_body(mail)).to match(translated(reportable.body))
+      end
+    end
+
+    describe "#hidden_by_admin" do
+      let(:mail) { described_class.hidden_manually(user, report, user) }
+
+      let(:mail_subject) { "Un contingut s'ha ocultat automàticament" }
```

Comment:
This subject is wrong. I understand that this because is checking the i18n key of an untranslated text (like it says in line 169), so the flow should be:

- merge this as is
- translate the new strings in Crowdin in catalan
- merge the decidim-bot PR 
- uncomment line 170 (and remove line 169) 
- fix the spec

Right?